include/vnode.h

```diff
@@ -83,28 +85,34 @@ typedef struct vattr {
 } vattr_t;
 #endif
 
+#define VOP_CALL(op, vp, ...) ((vp)->v_ops->v_##op)((vp), __VA_ARGS__)
```

Comment:
I don't like idea behind this macro. This doesn't introduce any value to the code except macro. Unfamiliar programmer will spend few extra seconds to see what it means, thus making it slightly more confusing. And I'm really used to old way of VOP functions.